README.md

```diff
@@ -4,52 +4,56 @@ At Recurrent, we work with data we collect from people’s electric vehicles
+  battery is depleted, expressed in miles as a decimal number.
+- `odometer`: The vehicle’s current odometer reading, expressed in miles as an
+  integer.
+- `plugged_in`: Whether or not the vehicle is plugged in at the time of the
+  reading, expressed as a boolean.
+- `charging`: Whether or not the vehicle is actively charging at the time of
+  the reading, expressed as a boolean.
+- `created_at`: Timestamp of the reading, expressed in a string of the format
+  "yyyy-mm-dd hh:mm:ss", the timezone is UTC for all readings.
+- `vehicle_id`: A unique identifier tied to the vehicle, expressed as a
+  string.
+
+The ev_data file in this repo contains a collection of data points over the
+course of a few weeks for 5 different vehicles. We provide the file in both
+CSV and JSON formats for your convenience since it's sometimes easier to work
+with one or the other in a given programming language. You may use whichever
+one you prefer, and your program DOES NOT need to work with both file types.
 
 ## Task 1: Write some code and a README
 
-Your first task is to write a command line program that reads the data in the
-csv and is able to execute these two queries.
+Your first task is to write a command line program that reads the data from the
+data file and is able to execute these two queries.
 
- 1. `charged_above`: This query should return the number of vehicles that
+1.  `charged_above`: This query should return the number of vehicles that
     reported at least one `charge_reading` above a given % over the whole time
     period. It should take a charge % as an argument. This argument should be a
     decimal, for example 0.33 will be passed to indicate 33%.
- 2. `average_daily_miles`: This query should return the average daily miles for
+2.  `average_daily_miles`: This query should return the average daily miles for
     a given vehicle over the course of the time period of the dataset, so it
     should take `vehicle_id` as an argument. (For example, if the given vehicle
     travelled 140 miles over a two week period, this should return 10).
 
 We would like to be able to execute the query by typing a command that executes
 your program by passing in three required arguments.
- 1. the path to the csv file for the data
- 2. the name of the query (specified in bold in the list above)
- 3. one argument to be passed to the query (a vehicle identifier, or a number,
+
+1.  the path to the data file
+2.  the name of the query (specified in bold in the list above)
+3.  one argument to be passed to the query (a vehicle identifier, or a number,
     as specified in the query description)
 
 For example, if the program you have written is in ruby, and the entry point
 file is named "ev_data_query.rb", then the command you would type in to execute
 query number 2 below is:
 
 ```
-ruby ev_data_query.rb /path/to/csv/file average_daily_miles cat-car
+ruby ev_data_query.rb /path/to/data/file.csv average_daily_miles cat-car
```

Comment:
I'm wondering if we should leave this generic regarding the file? Someone could choose to use the json file with ruby instead of the csv?

Reply:
Yep, makes sense. I was just trying to convey that it's okay for a file extension to be part of that path and that they don't need to magically append a ".csv" or ".json" inside of their program, but it's probably fine to omit it... or use a generic `.ext`...?

Reply:
I tweaked it to this, curious what you think:

> For example: if the program you have written is in ruby, the entry point
file is named "ev_data_query.rb", and you've chosen to work with the CSV
file, then the command you would type to execute query number 2 below is:
> 
> ruby ev_data_query.rb /path/to/data/file.csv average_daily_miles cat-car

Reply:
good point! I like this, and the fact that it calls it out as a very specific example!